### DISCUSSION TITLE
Trying to install WhatsApp, installer not moving forward.

### DISCUSSION THREAD

**Discussion by adityamwagh** [bottlesdevs/Bottles#586] [Q&A]
I am trying to install 64-bit WhatsApp for Windows. It is showing a popup asking me to install .NET, after which I click install, but the progress bar isn't moving forward after a point

![bottles_error](https://user-images.githubusercontent.com/25822972/135651048-e48ebe2a-ea4a-49a4-a240-954911b16f19.png)

What might be creating an issue here?

**Comment by mirkobrombin** [comment]
It is more a wine related problem then a Bottles one. Try intalling the .Net framework from the dependencies tab or wait until it finish.

**Reply by adityamwagh** [reply]
I waited for a whole afternoon. It still didn't move forward. I will try the method you suggested.

**Reply by AliYar-Khan** [reply]
@adityamwagh were you able to install it ?

**Reply by adityamwagh** [reply]
Sorry, I don't remember.
